src/db.py

```diff
@@ -0,0 +1,34 @@
+__author__ = 'Tsintsir'
+
+from binascii import b2a_hex
+from os import urandom
+from pymongo import MongoClient
+from pbkdf2 import PBKDF2
+
+
+def create_user(username, email, password):
```

Comment:
I've been working on gitolite integration here, if you want some example:

https://github.com/gitsubmit/gitsubmit-backend/blob/user/shawkins/gitolite_integration%2318/src/gitolite.py#L9

I will open a PR for this once I get everything squashed down and get tests written, but might be helpful in the meantime for inspiration